engineio/async/eventlet.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
```

Comment:
Is the __future__ thing really needed? This project does not run on anything below 2.7.